packages/styles/scss/patterns/blocks/contentgroupcards/_contentgroupcards.scss

```diff
@@ -66,6 +62,11 @@
       }
     }
 
+    .#{$prefix}--card-link__title,
+    .#{$prefix}--card-link__content {
```

Comment:
Something to keep in mind is I think @annawen1's pull request with her renaming of the card got merged yesterday.

```scss
.#{$prefix}--card__title,
.#{$prefix}--card__content { ... }
```